src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1154,7 +1169,7 @@ protected sealed override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownA
             }
         }
 
-        internal override bool IsExplicitDefinitionOfNoPiaLocalType
+        internal sealed override bool IsExplicitDefinitionOfNoPiaLocalType
```

Comment:
>IsExplicitDefinitionOfNoPiaLocalType [](http://example.com/codeflow?start=38&length=36)

Are we testing an attempt to make an extension type a NoPia local type?

Reply:
Retargeting is not in scope for this PR (see OP). I'm planning to address in PR that handles emit and PE.

Reply:
> Retargeting is not in scope for this PR (see OP). I'm planning to address in PR that handles emit and PE.

This isn't primarily related to retargeting. Consider adding a PROTOTYPE comment